bridge-ext/src/main/java/com/liferay/faces/bridge/ext/mojarra/spi/internal/FacesConfigResourceProviderLiferayImpl.java

```diff
@@ -36,8 +36,6 @@
 	 */
 	@Override
 	public Collection<URI> getResources(ServletContext servletContext) {
-		Collection<URI> facesResources = getResources("faces-config.xml");
```

Comment:
@juangon, @ngriffin7a,
I removed this, since Mojarra seems to be finding all the `faces-config.xml` files correctly in [`MetaInfFacesConfigResourceProvider`](https://github.com/javaserverfaces/mojarra/blob/master/jsf-ri/src/main/java/com/sun/faces/config/configprovider/MetaInfFacesConfigResourceProvider.java#L162). Also since both our code and Mojarra's code were both providing `faces-config.xml`, the `faces-config.xml` files were being scanned twice. It caused our render kits to be added to the delegation chain twice (which was how I discovered the bug), and it probably could cause other problems.